openmetadata-ui/src/main/resources/ui/src/utils/ToastUtils.ts

```diff
@@ -47,34 +47,36 @@ export const showErrorToast = (
   autoCloseTimer?: number,
   callback?: (value: React.SetStateAction<string>) => void
 ) => {
-  let errorMessage;
-  if (isString(error)) {
-    errorMessage = error.toString();
-  } else {
-    const method = error.config?.method?.toUpperCase();
-    const fallback =
-      fallbackText && fallbackText.length > 0
-        ? fallbackText
-        : i18n.t('server.unexpected-error');
-    errorMessage = getErrorText(error, fallback);
-    // do not show error toasts for 401
-    // since they will be intercepted and the user will be redirected to the signin page
-    // except for principal domain mismatch errors
-    if (
-      error &&
-      (error.response?.status === ClientErrors.UNAUTHORIZED ||
-        (error.response?.status === ClientErrors.FORBIDDEN &&
-          method === 'GET')) &&
-      !errorMessage.includes('principal domain')
-    ) {
-      return;
+  if (!axios.isCancel(error)) {
```

Comment:
@Ashish8689 , here we can early return if the error is type of `Cancel` interface.
```ts
if(isCancel(error)){
  return;
}